cylc/flow/data_messages.proto

```diff
@@ -108,8 +108,8 @@ message PbJob {
     optional string submitted_time = 6;
     optional string started_time = 7;
     optional string finished_time = 8;
-    optional string batch_sys_job_id = 9;
-    optional string batch_sys_name = 10;
+    optional string job_id = 9;
+    optional string job_runner_name = 10;
```

Comment:
@dwsutherland Can I get a check from you that this protobuf stuff has been done correctly?

Reply:
Looks good to me, as long as the instructions at the top were followed, the binary used to generate the module is the same version protobuf, and tests pass 👍

Reply:
The diff in the generated module looks appropriate.. Will re-generate on review and see if there are any changes 👍